src/utils/index.js

```diff
@@ -91,22 +91,20 @@ export const blockToDate = (goalBlock, height) => {
   return new Date(diff * 180000 + Date.now());
 };
 
-export const wrapTry = async (promise, store) => {
+// eslint-disable-next-line consistent-return
+export const wrapTry = async (promise) => {
   try {
     return promise.then((res) => {
       if (!res) {
-        if (store) { store.commit('setBackendStatus', false); }
-        return null;
+        throw new Error('No response object');
       }
-      if (store) { store.commit('setBackendStatus', true); }
-      if (!res.ok) throw new Error(`Request failed with ${res.status}`);
+      if (res.status !== 200) {
+        throw new Error(`Error code ${res.status}`);
+      }
+      if (!res.ok) throw new Error(4000);
```

Comment:
Why 4000?? I would just leave as it was before, `res.ok` check should be enough, and "Request failed with" looks better